config/branch/master.ini

```diff
@@ -6,3 +6,14 @@ deploy_url = ${dev_deploy_repo_url}/${repo}
 
 # Master images should always follow the master ostree.
 stable_branch = ${build:branch}
+
+[flatpak-remote-flathub]
+# Don't install the stable version of Kolibri
+apps_del =
+  org.learningequality.Kolibri
+
+[flatpak-remote-flathub-beta]
+# Instead, install the beta version of Kolibri
+repo_file = https://dl.flathub.org/beta-repo/flathub-beta.flatpakrepo
```

Comment:
Even though master is probably the only place we'd ever use the flathub beta repo, I'd prefer if it was defined in `defaults.ini` with `enable = false`. Like:
```
[flatpak-remote-flathub-beta]
repo_file = https://dl.flathub.org/beta-repo/flathub-beta.flatpakrepo
# Disabled by default
enable = false
```
Then in `master.ini` just add `enable = true`.

But I would actually split that into a separate commit since enabling the beta repo now means it's available all the time. Ideally we would set `xa.prio` to 0 (or maybe even `xa.noenumerate` to `true`, but I'm not totally sure what the affects) so that you still use the normal flathub repo by default. But that would require some more plumbing. Maybe it's OK if master images are setup to prefer flathub beta releases at runtime.

Reply:
Oh yeah, that's tidier. Thanks! :)